src/api/wrappers/cf_wrapper.py

```diff
@@ -20,45 +20,64 @@ class CurseForgeWrapper:  # pylint: disable=too-few-public-methods
     def __init__(self, api_key):
         self.api_key = api_key
 
-    def get_mod(self, mod_id: int) -> CFDataResponse:
+    def _get_headers(self) -> dict:
+        return {
+            'Accept': 'application/json',
+            'Content-Type': 'application/json',
+            'x-api-key': self.api_key
+        }
+
+    def _request(self, path: Path, body: dict = None) -> dict:
+        try:
+            url = urljoin(CF_BASE_URL, Path(CF_API_VERSION) / str(path))
```

Comment:
`Path(CF_API_VERSION) / str(path)` returns a Path object, where `urljoin()` expects a string. Convert to string before feeding to `urljoin()`